templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,12 +13,14 @@
    "cell_type": "markdown",
```

Comment:
<p>It's not clear from the directions which cells the user needs to modify, and which can be run as-is. I would try and group together all of the lines that don't need modification into a smaller number of cells to reduce cognitive load on the user. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>